tox.ini

```diff
@@ -42,12 +42,12 @@ deps =
 
     postgres:
         psycopg2>=2.9
-        .[pgvector]
```

Comment:
@mgax adding `package = editable` to the `[testenv]` section achieves the same effect and is probably more desirable from the tox point of view.

See https://tox.wiki/en/4.13.0/config.html#use_develop / https://tox.wiki/en/4.13.0/config.html#package

Reply:
@zerolab thanks for the pointers, that's much nicer. Unfortunately it doesn't seem to work. I suspect that listing `.` in the dependencies clashes with this option. I didn't have much luck breaking out the extras, couldn't get tox to install the _testing_ extra dependencies, even though this seems the way to do it:

```ini
[testenv]
extras = testing
```

---
In any case, for my needs right now, I'm running _pytest_ outside of tox, and it's fine.

Reply:
oh, that's a bummer. 
This should be OK then.